app/views/idv/doc_auth/send_link.html.erb

```diff
@@ -0,0 +1,39 @@
+<% title t('doc_auth.titles.doc_auth') %>
+
+<% if flow_session[:error_message] %>
+  <div class='alert alert-error'>
+    <%= flow_session[:error_message] %>
+  </div>
+<% end %>
+
+<h1 class='h3'>
+  <%= t('doc_auth.headings.take_picture') %>
+
+<p class='mt-tiny mb3'><%= t('doc_auth.info.take_picture') %></p>
+
+<p class='mt-tiny mb3 bold'><%= t('doc_auth.info.camera_required') %></p>
+
+<p class='mt-tiny mb3'><%= t('doc_auth.instructions.send_sms') %></p>
+
+<%= simple_form_for(:doc_auth, url: url_for, method: 'PUT',
+        html: { autocomplete: 'off', role: 'form', class: 'mt2' }) do |f| %>
+  <div class='clearfix mxn1'>
+    <div class='sm-col sm-col-8 px1'>
+      <!-- using :tel for mobile numeric keypad -->
```

Comment:
this comment seems like a like...maybe it meant `:phone` ? I have a feeling it's my comment

Reply:
I can change while we're here